app/src/main/java/com/hajduk/systems/prepareordermailing/adapter/woocommerce/WooCommerceClient.kt

```diff
@@ -49,17 +49,19 @@ class WooCommerceClient(
     }
 
     private fun <T : Any> getResource(resourceUrl: String, clazz: Class<T>, onSuccess: (data: T) -> Unit, onNotFound: () -> Unit, onFailure: (message: String) -> Unit) {
-        val signedUrl = createSignedUrl(resourceUrl)
-        Log.d(LOGGING_TAG, "Getting resource $resourceUrl, url: $signedUrl")
-        Fuel.get(signedUrl)
+        Fuel.get(createSignedUrl(resourceUrl))
+            .timeout(5000)
+            .timeoutRead(10000)
```

Comment:
10 sekund? Może pomyślimy nad jakimś spinnerem

Reply:
Jest na to zadanie przypisane do Ciebie :D